ogr/services/github.py

```diff
@@ -57,6 +57,38 @@
 logger = logging.getLogger(__name__)
 
 
+class GithubRelease(Release):
+    project: "GithubProject"
+
+    def __init__(
+        self,
+        title: str,
+        body: str,
+        tag_name: str,
+        url: str,
+        created_at: str,
+        tarball_url: str,
+        git_tag: GitTag,
+        project: "GithubProject",
+        raw_release: PyGithubRelease,
+    ) -> None:
+        super().__init__(
+            title, body, tag_name, url, created_at, tarball_url, git_tag, project
+        )
+        self.raw_release = raw_release
+
+    def edit_release(self, name: str, message: str) -> None:
+        """
+        Edit name and message of a release.
+
+        :param name: str
+        :param message: str
+        """
+        self.raw_release.update_release(name=name, message=message)
+        self.title = self.raw_release.title
+        self.body = self.raw_release.body
```

Comment:
I have just one idea about this. It would be usefull to use properties for that so you do not need to update the attributes:

```suggestion
        
    @property
    def body(self):
        return self.raw_release.body
```

(Same for the `self.title` and other attributes.)

I am just worried, that we need to remove these atributes from the `__init__` method of the `Release` and create abstract properties for each of them in the `Release` instead:

```python
class Release:

    def __init__(self):
        pass

    @property
    def body(self):
        raise NotImplementedError()
```

Not sure if it is worth it...